apps/web/src/components/SearchRequestTable/SearchRequestsTableApi.tsx

```diff
@@ -255,18 +287,19 @@ const SearchRequestsTableApi = ({
       },
       {
         label: intl.formatMessage({
-          defaultMessage: "Request job title",
-          id: "6AhLsc",
+          defaultMessage: "View Request",
+          id: "kZGKwO",
           description:
-            "Title displayed for the search request table request job title column.",
+            "Title displayed for the search request table view column.",
         }),
-        id: "job_title",
-        sortColumnName: "job_title",
+        id: "view_request",
+        sortColumnName: "view_request",
```

Comment:
I think we will need to remove this since it is a virtual column that we technically can't sort on through the API.